build.gradle

```diff
@@ -28,6 +28,19 @@ plugins {
   id "net.ltgt.errorprone" version "3.0.1" apply false
   id 'com.diffplug.spotless' version "6.5.2" apply false
   id 'com.github.node-gradle.node' version '3.4.0' apply false
+  id 'org.cyclonedx.bom' version '1.7.2'
+}
+
+cyclonedxBom {
+  includeConfigs = ["runtimeClasspath"]
+  skipConfigs = ["compileClasspath", "testCompileClasspath"]
+  projectType = "application"
+  schemaVersion = "1.4"
+  destination = file("build/reports")
+  outputName = "CycloneDX-Sbom"
+  outputFormat = "all"
+  includeBomSerialNumber = true
+  componentVersion = "2.0.0"
 }
```

Comment:
This should end up in a separate config file like `gradle/validation/cyclonedx.gradle` - another example of similar config would be any of the files in `gradle/validation`